lib/matplotlib/text.py

```diff
@@ -123,6 +123,7 @@ def __init__(self,
                  linespacing=None,
                  rotation_mode=None,
                  usetex=None,          # defaults to rcParams['text.usetex']
+                 override_math=False,
```

Comment:
I'm not sure about the name, but regardless, this must go to the end of the argument list.

Reply:
I agree about putting it at the end and it should be made key-word only.


I suggest changing the name to something positive (like "parse_math", "interpret_math", "as_math") so that the default value can be `True` and then setting it to False when we do not want it.  In general it is better to avoid names / defaults that result in double negatives.  "(do|do not) not try to parse as math" is harder for people to understand than "(do|do not) try to parse as math"!

Reply:
ah, that makes sense!
let me update it with something like `parse_math`.

> should be made key-word only.

I didn't quite understand this?

Reply:
Add a `*`: https://docs.python.org/3/glossary.html#term-parameter

Reply:
> Add a `*`: https://docs.python.org/3/glossary.html#term-parameter

Explains so many class' __init__s, going to use it for the first time!